### PR TITLE
Sparkle: Adding new icons

### PR DESCRIPTION
**Pull request by pinotalexandre**
## Description
Adding new icons to Sparkle:
- TSquare
- Camera

## Risk
First PR for me, so maybe pay more attention in case I made any mistakes.

## Deploy Plan
- Merge
- Publish sparkle

### CODE REVIEW

File: sparkle/package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
     "react-confetti": "^6.1.0",
     "react-dropdown-menu": "^0.0.2",
     "react-markdown": "^9.0.1",
+    "shadcn-ui": "^0.9.3",
```
Comment (aubin-tchoi) [inline sparkle/package.json:115]: OOC: how come do we need this as a dependency?

File: sparkle/src/icons/src/solid/TSquare.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (aubin-tchoi) [inline sparkle/src/icons/src/solid/TSquare.svg:1]: OOC: to which SVGs do we allow/disallow passing props? (I don't know much about which props are used for svg besides the color)

File: sparkle/package.json (hunk 1 above)
Comment (pinotalexandre) [reply sparkle/package.json:115]: @Duncid I guess that you'll be able to answer

**Review by JulesBelveze — APPROVED** [review]
Looking good man 🔥

**Pull request merged**